NDB.Covid19/NDB.Covid19.Droid/Resources/values/strings.xml

```diff
@@ -3,6 +3,4 @@
   <string name="app_name">Smittestopp</string>
   <string name="action_settings">Settings</string>
   <string name="package_name">no.fhi.smittestopp_exposure_notification</string>
-  <string name="channel_name">Local Notifications</string>
```

Comment:
Where did you move this one?

Reply:
This was used for all notifications but is now deleted, and all the new ones are added to the locales